Задание_8.5/Задание_8.5/Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+
+namespace Задание_8._5
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите последовательность чисел");
```

Comment:
Сообщение, вводящее в заблужение
```suggestion
            Console.WriteLine("Введите целое число");
```